gunicorn/arbiter.py

```diff
@@ -340,9 +340,12 @@ def wakeup(self):
     def halt(self, reason=None, exit_status=0):
         """ halt arbiter """
         self.stop()
-        self.log.info("Shutting down: %s", self.master_name)
+
+        log_func = self.log.info if exit_status == 0 else self.log.error
+        log_func("Shutting down: %s", self.master_name)
         if reason is not None:
-            self.log.info("Reason: %s", reason)
+            log_func("Reason: %s", reason)
```

Comment:
shouldn't we keep the line "Shutting down ..."  ?

Reply:
I did. It moved above the if statement so that it's always logged.

Reply:
Oh, sorry, it was always logged in both cases. I moved it down so that it logs at error level when the status is non-zero.